Makefile

```diff
@@ -21,8 +21,8 @@ build: clean install lint css minify
 	echo "" && \
 	echo "Site built out to ./public dir"
 
-serve: install lint js css minify
-	$(PREPEND)$(NPMBIN)/hugo server
```

Comment:
why change this?

Reply:
Serving the site without rendending future blog posts makes it not very useful when testing. I can change it back though.